Lib/test/test_typing.py

```diff
@@ -4924,12 +4924,17 @@ def test_valid_uses(self):
 
     def test_args_kwargs(self):
         P = ParamSpec('P')
+        P_2 = ParamSpec('P_2')
         self.assertIn('args', dir(P))
         self.assertIn('kwargs', dir(P))
         self.assertIsInstance(P.args, ParamSpecArgs)
         self.assertIsInstance(P.kwargs, ParamSpecKwargs)
         self.assertIs(P.args.__origin__, P)
         self.assertIs(P.kwargs.__origin__, P)
+        self.assertEqual(P.args, P.args)
+        self.assertEqual(P.kwargs, P.kwargs)
+        self.assertNotEqual(P.args, P_2.args)
+        self.assertNotEqual(P.kwargs, P_2.kwargs)
```

Comment:
Just some extra combinations:
- `P.args` and `P.kwargs`
- `P.kwargs` and `P.args` (this is important, since we use `NotImplemented`)
- `P.args` and `P_2.kwargs`

Reply:
That's a nice way to cover `NotImplemented`. Added!